spec/src/debugger-spec.coffee

```diff
@@ -27,7 +27,7 @@ describe 'ProcessManager', ->
         config:
           get: (key) -> mapping[key]
 
-      spyOn(childprocess, 'spawn').andReturn(makeFakeProcess())
+      spyOn(childprocess, 'spawn').andReturn(Q.fcall(makeFakeProcess))
```

Comment:
Q uses a method called 'progress' which does not seem to exist in the bluebird implementation (or it is called something else) so this does not work. :(